Plugins/Flow.Launcher.Plugin.WebSearch/SuggestionSources/Baidu.cs

```diff
@@ -22,9 +23,9 @@ public override async Task<List<string>> Suggestions(string query)
             try
             {
                 const string api = "http://suggestion.baidu.com/su?json=1&wd=";
-                result = await Http.Get(api + Uri.EscapeUriString(query), "GB2312");
```

Comment:
removal of "GB2312" is intentional?

Reply:
Yes, I have checked, the ReadAsString doesn't need the encoding to create the result.

Reply:
do we need to keep  "GB2312"?